Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -21,7 +21,7 @@ function HandleCodeAnalysisReporting
     }	
     else
     {
-        Write-Verbose "The build was not triggered by a Pull Request, not processing code analysis comments"   
+        Write-VstsTaskVerbose "The build was not triggered by a Pull Request, not processing code analysis comments"   
```

Comment:
*I think* if you change it to always `Write-VstsTaskVerbose`, then you are forcing the debug output regardless of whether system.debug is set.